packages/ppxfind/ppxfind.1.4/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Tool combining ocamlfind and ppx"
+description: """
+Ppxfind is a small command line tool that among other things allows
+to use old style ppx rewriters with jbuilder.
+"""
+maintainer: ["Jérémie Dimino <jeremie@dimino.org>"]
+authors: ["Jérémie Dimino <jeremie@dimino.org>"]
+license: "BSD3"
+homepage: "https://github.com/diml/ppxfind"
+doc: "https://diml.github.io/ppxfind/"
+bug-reports: "https://github.com/diml/ppxfind/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml-migrate-parsetree"
```

Comment:
```suggestion
  "ocaml-migrate-parsetree" {>= "1.6.0"}
```
@aantron is that what you were saying in #15821 ?

Reply:
Yes, but I think we need to merge that PR first, before the change would make sense (the tests shouldn't be able to solve the constraints until then). See also https://github.com/diml/ppxfind/commit/14166274726ca24db097cbe18066ab9b2b76d9ea#diff-4f99a9f8e368c141259cd296f03c6049L15.

Reply:
BTW, I tested the build with omp 1.5.0 when I submitted the PR and it was building fine.

Reply:
So I'm not sure the constraint is necessary at all. Except if it's for some other reason I missed.

Reply:
The package itself compiled with omp 1.5.0 but the revdeps came back all negative: https://ci.ocaml.org/ocaml/opam-repository/pr/15830?history=5791b5da8eaa2ab041e18fd9eac2e1f22344341c&test=2%20Revdeps

Reply:
It's necessary, because the `~argv` support is buggy before 1.6.0. I fixed it in https://github.com/ocaml-ppx/ocaml-migrate-parsetree/commit/0d28ffb24a3e555f280971fc6672ef39da439f5a, that's what parsing from the first argument refers to.

Reply:
The "full" history is that I added `~argv` support for a specific use case in Bisect_ppx, that wasn't released anyway. The `~argv` support was released in 1.5.0, which was released for other reasons not related to Bisect_ppx or any other project that needed `~argv` at the time. It just happened to already be in `master` :) I realized that the support was buggy almost immediately, but the PR that fixed that remained unmerged (by me) for several months, across the 1.5.0 release, because, again, there was no pressing need for the feature anyway. I didn't even know there was a 1.5.0 release for a while :p 1.6.0 is the first release that properly includes the working version of this feature.

Reply:
The release then broke ppxfind, which I wasn't aware of, but `~argv` was the way to fix ppxfind. The buggy version of `~argv` in 1.5.0 was buggy in exactly the right way to not break ppxfind.